### DISCUSSION TITLE
[CN DateTimeV2] Java support in Chinese

### DISCUSSION THREAD

**Discussion by codemayq** [microsoft/Recognizers-Text#1329] [General]
Thanks for your work a lot.
I want to known if any official plan to support the DateTime of Chinese version in Java,  because I haven't seen it yet.
If not any plan,  I want to be a contributor of DateTime of Chinese version in Java and will start to read and push code requests recently. 
I used to be an intern in MSRA SA group , also a user of this project. Now I has some needs of Chinese version in Java personally for my master thesis, and I also want to do some things helpful for this repo.

Thanks.

**Comment by tellarin** [comment]
@codemayq sorry for the delay to reply. Yes, it is coming soon.
It would be great if you could contribute! Let me know if you need any help and we can discuss steps by email/Skype.
Thanks!

**Comment by pursue-wind** [comment]
> @codemayq sorry for the delay to reply. Yes, it is coming soon.
> It would be great if you could contribute! Let me know if you need any help and we can discuss steps by email/Skype.
> Thanks!

Thanks for your work a lot. 
How long does it take for Java to support datetime?

**Comment by tellarin** [comment]
@codemayq, are you still interested to contribute to it?
